src/main/java/com/antelif/library/application/controller/command/TransactionCommandController.java

```diff
@@ -52,7 +52,7 @@ public ResponseEntity<Map<String, TransactionResponse>> createTransaction(
    * @param bookCopyIds the ids of the book copies that are returned,
    * @return a list of updated transaction response DTOs.
    */
-  @PatchMapping("/customer/{customerId}")
+  @PatchMapping(TRANSACTIONS_ENDPOINT + "/customer/{customerId}")
```

Comment:
It's not in the scope of this ticket. But I think this enpoint should change to something more descriptive.